### DISCUSSION TITLE
Env var required but not provided

### DISCUSSION THREAD

**Discussion by vymaker** [AltimateAI/vscode-dbt-power-user#1192] [Q&A]
I get this error  for vars that have been defined in `.bashrc`
```
Parsing Error
  Env var required but not provided.... 
```

I can run dbt commands in the vscode integrated terminal without any issue, but dbt-power-user complains on missing env var. Any idea? Thanks a lot in advance!

- dbt-databricks==1.5.7
- on 2 platforms: windows + git bash, and github codespace/linux

I have the same setup with `.zshrc`on mac, and it works fine there.

**Comment by anandgupta42** [comment]
Please have a look at https://docs.myaltimate.com/setup/optConfig/#environment-variables-setup

You can also run print env command in the extension to get the list of detected env variables

**Reply by vymaker** [reply]
I had already gone through that before writing here. But never mind, I figured it out, and I think it should be added to the docs as well. To sum up:
- In windows, VS Code should be opened from git bash terminal to pass env vars but not as a desktop application ❗ 
- With Github Codespace, secrets should be defined in [settings](https://docs.github.com/en/codespaces/developing-in-a-codespace/persisting-environment-variables-and-temporary-files#use-development-environment-secrets) now though it used to work via `.bashrc` previously.

Thanks for taking your time to answer my questions, appreciate it!
